shell/src/main/java/org/apache/accumulo/shell/commands/ImportDirectoryCommand.java

```diff
@@ -29,21 +29,33 @@
 
   @Override
   public String description() {
-    return "bulk imports an entire directory of data files to the current"
-        + " table. The boolean argument determines if accumulo sets the time.";
+    return "bulk imports an entire directory of data files into an existing table."
+        + " The table is either passed with the -t tablename opt, or into to the current"
+        + " table if the -t option is not provided. The boolean argument determines if accumulo sets the time.";
   }
 
   @Override
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
       throws IOException, AccumuloException, AccumuloSecurityException, TableNotFoundException {
-    shellState.checkTableState();
+
+    boolean tableOptPassed = cl.hasOption(OptUtil.tableOpt().getOpt());
```

Comment:
I think `OptUtil.getTableOpt()` is supposed to do most of this logic. As in:

```java
  String tableName = OptUtil.getTableOpt(cl, shellState);
```

Either way, you also need to declare it as an option in the `getOptions` method, something like:

```java
  @Override
  public Options getOptions() {
    Options o = new Options();
    o.addOption(OptUtil.tableOpt("name of the table to import files into"));
    return o;
  }
```